oop_Session1/CompositeElement.java

```diff
@@ -0,0 +1,54 @@
+package oop_Session1;
+
+import java.util.List;
+import java.util.ArrayList;
+/**
+ * This class is responsible for initializing Composite element i.e. the elements which have more tags inside it. 
+ * @author Akshita Dixit
+ */
+public class CompositeElement extends Element{
+	
+	// List stores tags inside parent tag
+	List<Element> element = new ArrayList<Element>();

```

Comment:
Variables of "collection" types (like lists, sets, arrays, etc.) should be named in the plural: `List<Element> elements`